modAionImpl/src/org/aion/zero/impl/AionHub.java

```diff
@@ -248,7 +248,7 @@ private void loadEventMgr(boolean forTest) {
         prop.put(EventMgrModule.MODULENAME, "org.aion.evtmgr.impl.mgr.EventMgrA0");
         try {
             this.eventMgr = EventMgrModule.getSingleton(prop).getEventMgr();
-        } catch (Throwable e) {
+        } catch (Exception e) {
             genLOG.error("Can not load the Event Manager Module", e.getMessage());
```

Comment:
additional correction needed here: `getMessage()` may sometime be null. I think it's best to put the whole exception when logging.